app/javascript/components/Articles/ArticlesList.tsx

```diff
@@ -0,0 +1,40 @@
+import React, { useState, useEffect } from "react";
+import ArticlesDetails from './ArticlesDetails';
+import { array } from "prop-types";
+
+export interface ArticlesListProps {
+    link?:string
+}
+
+export default function ArticlesList(props) {
+
+    let display = (<h3>It looks as though there's nothing here yet!</h3>)
+
+    function formatDate(date) {
+        let foo = date.toString().split('T')[0]
+        let bar = foo.split('-')
+        return bar[2] + "-" + bar[1] + "-" + bar[0] 
+    }
+
+    if (props.articles) {
+        display = <div>
+            {props.articles.map((article, index) => 
+                <ArticlesDetails 
+                key={index} 
```

Comment:
I think in general this is considered bad practice. This article gives a good idea as to why https://medium.com/@vraa/why-using-an-index-as-key-in-react-is-probably-a-bad-idea-7543de68b17c

Reply:
Thanks for pointing this out!